packages/terra-application-navigation/src/ApplicationNavigation.module.scss

```diff
@@ -59,18 +59,18 @@
   }
 
   [dir=ltr] .content-layout.drawer-menu-is-open {
-    transform: translate3d(var(--terra-application-navigation-content-layout-default-width, calc(100% - 3.571rem)), 0, 0);
```

Comment:
Shouldn't these values have been removed from the theme files?

Reply:
Or should these theme values been updated to:
```
transform: var(--terra-application-navigation-content-layout-transform, translate3d(calc(100% - 3.571rem), 0, 0));
```

Reply:
>Shouldn't these values have been removed from the theme files?

These variables are not present in the theme files.

@emilyrohrbough I tried how you have suggested, but it didn't work. What I think the issue with `calc` is that when it's used inside any function IE does not recognize it. Whereas if it's used as outer function its works as expected.

Reply:
@pranav300 yeah Tyler mentioned the issue was with calc being within transform, not necessarily because fo the theme var.